portfolio-website/script.js

```diff
@@ -1,23 +1,79 @@
+// navBar
 const navBarList = [
-    "Ingredients", "Steps to Make", "Links to Other Wonton Soup Recipes", "About Me"
+    "About Me", "Projects", "Contact"
 ]
 
 for (let i = 0; i < navBarList.length; i++) {
     const label = document.createElement('a');
     label.innerText = navBarList[i];
```

Comment:
we have accessed navBarList[i] two times here, if we just assign to variable then re-use it, it would help w readability. ex. 

```
const currNavText = navBarList[i];

label.innerText = currNavText;
...
...

label.setAttribute('href', `#${currNavText.split("").join("")}`);


```